### DISCUSSION TITLE
你们未来会推出端侧向量数据库版本吗？

### DISCUSSION THREAD

**Discussion by dushulin** [milvus-io/milvus#29215] [Q&A and General discussion]
是否考虑在端侧部署milvus，或者有哪些端侧向量库推荐？

**Comment by yhmo** [comment]
如果是在移动设备上，数据量大了也不好处理。不考虑数据一致性安全性这些的话，手写一个简易的工具也不难，但没有门槛，不适合商业化。还不如手机上的app直接向云上的向量数据库发restful来得方便。
如果是用在边缘计算，目前的milvus确实太重，如果加以瘦身改造，是有可能做到的，但这不在我们目前的主路径上。
